marpa/lib/standard.pm

```diff
@@ -21,5 +27,31 @@ sub import ( $class ) {
     return;
 }
 
+sub strip_terminators ($contentref) {
+    ${$contentref} =~ s/^__DATA__\n.*//xms;
+    ${$contentref} =~ s/^__END__\n.*//xms;
```

Comment:
I understood from a previous comment of yours that the idea was to remove everything **between** `__DATA__` and `__END__`, but I don't think this code does that? Of course, removing everything after `__END__` seems fine.

Reply:
You're right. I thought about doing what I suggested but when I sat to write the code, it didn't really make sense to remove chunks. Everything after either one is useless for the parser, so I'm just eliminating it.